src/App.css

```diff
@@ -0,0 +1,179 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+
+.hidden {
+  display: none;
+}
+
+nav {
+  width: 100%;
+  height: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  background-color: #ec4c8a;
+}
+
+nav li {
+  width: 25px;
+  height: 25px;
+  list-style: none;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  cursor: pointer;
+}
+
+#menu {
+  background-image: url(../src/components/menu.png);
+}
+
+nav > p {
+  color: #fff;
+}
+
+nav > ul {
+  display: flex;
+  width: 75px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#back {
+  background-image: url(../src/components/back.png);
+  margin-left: -7vw;
+}
+
+#mic {
+  background-image: url(../src/components/mic.ico);
+}
+
+#settings {
+  background-image: url(../src/components/settings.png);
+}
+
+#search {
+  background-image: url(../src/components/search.png);
+}
+
+#search-bar {
+  float: right;
+  outline: none;
+  border: none;
+  border-radius: 5px;
+  padding: 5px;
+  position: fixed;
+  right: 0;
+  top: 15px;
```

Comment:
- [ ] You did well implementing the search bar however the position still need some adjustment, I recommend changing the padding to 8px, top of 10 px and right 8px for example.


<img width="492" alt="Screenshot 2023-08-02 at 10 47 08 PM" src="https://github.com/hmunish/react-metrics/assets/53347966/af8b7df1-1842-4a8d-a4bd-d9e051cc99ee">